Kitodo/src/main/java/org/kitodo/production/services/command/KitodoScriptService.java

```diff
@@ -203,13 +204,13 @@ private boolean executeOtherScript(List<Process> processes, String script)
                 copyDataToChildren(processes, script);
                 break;
             default:
-                return executeRemainingScript(processes, script);
+                return executeRemainingScript(processes);
```

Comment:
Change is not related to the issue. Please revert this and open a separate pull request for this change.